ci/jjb/jobs/pulp3_PR_pulp-smash.yaml

```diff
@@ -0,0 +1,256 @@
+- job:
+    name: 'pulp-smash-pulp3-pr'
```

Comment:
Please make this template and its name match in some way. From the readme:

> - Templates go in in files named after to the jobs they create, so they can be
>   easily found.
> - Jobs using those templates are defined in `jobs/projects.yaml`, and the name
>   of the project should match the name of the file containing that project's
>   jobs.

Also, can this file follow the naming convention used by other files? Use dashes, not a mix of dashes and underscores; use lower-case letters; place a dash between "pulp" and "3". How about naming this file `pulp-3-pr-pulpcore.yaml`?